CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
 
 option(GC_LEGACY_ENABLE ON)
 option(GC_TEST_ENABLE "Build the tests" ON)
+option(GC_DEV_LINK_DYN_LLVM "Link dynamic libraries of LLVM and MLIR. For developers only. Do not use it in packing the library." OFF)
```

Comment:
Suggest to change the option name to `GC_DEV_LINK_LLVM_DYLIB` which is more align with LLVM's `LLVM_LINK_LLVM_DYLIB`?

Reply:
I agree with you. I will change that.